### PR TITLE
MIDUWEB-483: Add 'find abo' case

### PR DESCRIPTION
**Pull request by Nilsilino**

### CODE REVIEW

File: src/es/components/molecules/abonnements/Abonnements.js [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+            </ks-o-tile-list>
+          ` : /* html */ `
+            <ks-m-tile namespace="tile-default-" data='${JSON.stringify(abonnement)}'>
+            </ks-m-tile>
+          `
+        ),
+      '')
+    : ''
+
+    return this.fetchModules([
```
Comment (Weedshaker) [inline src/es/components/molecules/abonnements/Abonnements.js:135]: do you need all these modules here?

**Review by Weedshaker — APPROVED** [review]

File: src/es/components/molecules/abonnements/Abonnements.js (hunk 1 above)
Comment (Nilsilino) [reply src/es/components/molecules/abonnements/Abonnements.js:156]: Removed unnecessary Imports

File: src/es/components/molecules/abonnements/Abonnements.js [hunk 2]
```diff
@@ -102,19 +107,35 @@ export default class Abonnements extends Shadow() {
...
         click-no-toggle-active
       >
         <span>${this.linkLabel}</span>
-      </ks-a-button>
+      </button>
     `
+    return this.fetchModules([
+      {
+        path: `${this.importMetaUrl}../../web-components-toolbox/src/es/components/atoms/button/Button.js`,
+        name: 'a-button'
```
Comment (Weedshaker) [inline src/es/components/molecules/abonnements/Abonnements.js:122]: ks-a-button vs a-button vs button?
siehe Zeile 110

**Review by Weedshaker — CHANGES_REQUESTED** [review]
Does it work with <button>?
And if yes, does the Abonnements.js style it itself? Then it wouldn't need that import...

**Pull request merged**